Rakefile

```diff
@@ -12,7 +12,7 @@ desc 'Run RSpec'
 # rubocop:disable Metrics/BlockLength
 namespace :spec do
   task all: [:main,
-             :rails, :railsredis, :railssidekiq, :railsactivejob,
+             :rails, :railsredis, :railsactivejob,
```

Comment:
`:railssidekiq` had [one file](test/contrib/rails/rails_sidekiq_test.rb) with no tests.

All relevant Rails+Sidekiq related tests have been merged with the `:railsactivejob` task.